cis194/week3/tien/homework3/test/GolfSpec.hs

```diff
@@ -19,3 +19,8 @@ main =
         localMaxima [2, 9, 5, 6, 1] `shouldBe` [9, 6]
         localMaxima [2, 3, 4, 1, 5] `shouldBe` [4]
         localMaxima [1, 2, 3, 4, 5] `shouldBe` []
+
+-- Exercise 3 --
+      it "should return correct histogram" $ do
+        histogram [1, 3, 2, 4, 1, 2] `shouldBe`
+          "0|| \n1|| **\n2|| **\n3|| *\n4|| *\n5|| \n6|| \n7|| \n8|| \n9|| \n"
```

Comment:
is this output correct? it seems rotated.

Reply:
lol will need to find a way to rotate it...

Reply:
Will transpose give you something close?